boofuzz/primitives/group.py

```diff
@@ -32,41 +33,20 @@ def __init__(self, name, values, default_value=None):
         for val in self.values:
             assert isinstance(val, (six.binary_type, six.string_types)), "Value list may only contain string/byte types"
 
-    @property
-    def name(self):
-        return self._name
+    def encode(self, value, mutation_context):
+        return self._render(value)
```

Comment:
`self._render()` seems not be implementated. I did't find it in `Group` or its parent.

Reply:
Group is another one I didn't functional test; I'll come back to it when I update the unit tests.